builtin/logical/mssql/path_config_connection.go

```diff
@@ -0,0 +1,88 @@
+package mssql
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfigConnection(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "config/connection",
+		Fields: map[string]*framework.FieldSchema{
+			"connection_params": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "DB connection parameters",
+			},
+			"max_open_connections": &framework.FieldSchema{
+				Type:        framework.TypeInt,
+				Description: "Maximum number of open connections to database",
+			},
```

Comment:
You mentioned about `verify_connection` in the comments but the changes are not here. Did you miss out on pushing the changes?